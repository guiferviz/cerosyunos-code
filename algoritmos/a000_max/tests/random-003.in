10
33
487399732 1028813596 562472360 2083076913 1428590641 38974327 458600333 2126903188 1894699584 643966906 605657474 190577407 1706009811 1791298239 2133755160 797289489 1383297056 79751546 805415477 933460584 1677976584 17916977 1284304861 529131491 311159697 244223970 1797132511 66524909 1916662689 339795320 1733722773 1765752985 241803167
33
658205277 86755744 817319294 496236127 1987001477 602145164 1869402843 1599629689 1769344212 41820575 1053913981 1401026899 931360200 1372800924 574572668 553135986 1449942751 266035193 1276843995 2123302199 1699943457 444090021 454301822 341300623 1624740502 1605272341 1407693488 1997003128 206546794 514645768 696186697 2092834776 900297179
33
1444289249 1450006210 984518912 339971645 1180037762 133255412 1718242966 83126464 1374549887 358165411 1997632515 1683050043 1448138333 853789956 1956275307 1905864936 709726145 1272338482 382876213 855962501 952871580 738681018 831989857 943671211 753780542 259091220 993436089 684416256 1275419724 1223331725 1887605499 1120975444 1676283341
33
2146155143 1857131135 476877978 896823368 442441219 2146208368 1086986230 334771499 527617675 1339819327 1684799297 159808323 936707244 1754032422 1392100999 896144935 952581553 1559010121 2015581148 721440768 433941651 1993430111 425394145 528972528 2135215484 309624606 1031595170 1170951867 1993356366 1520248807 1436983387 990201737 725375047
33
86750241 1093465510 840502436 310922231 283853207 1196033593 151585510 225184605 1032086848 527678353 992568294 444005443 2102335752 2058304605 407637880 1908090966 1701063478 1961806390 896342183 629761918 1308559422 1180656145 1357024473 436758659 1861584442 1532892607 1240380704 1823779247 2003036967 131323958 867388484 692408950 1294387031
33
1778616776 1694779325 1664415063 230398517 296805470 2052304939 489034867 118375452 1194960647 2030536163 1206018449 1918283775 1374068001 1461042745 25262450 1443856417 1800989510 1738560020 610416568 1984788837 1088395519 868302945 770811972 2092640521 967301698 1521241417 984322100 545570025 715832797 285597438 249440325 1629093127 157149924
33
1175999804 2028059341 1427893867 1687824797 149837947 1455960669 1706962610 1039904142 2118219520 1646653087 1208010313 1109905172 16329431 665297174 1907004821 1951571751 1410213066 1679411997 998187921 1446649061 1421125260 841928319 644449214 687948817 622333499 500300465 851274257 1120678069 458553152 2047192220 1151231907 2091471260 1965483227
33
987494148 1532953400 107249067 1014366542 1527862742 1125762112 65539699 102787842 222778551 1539365037 1480647358 344207367 1918287303 659650896 125567681 1903046873 467593616 1234407204 557035423 758666414 442614541 1188668766 1807286361 230087496 96077148 538484406 1176990077 1352130817 710979992 712389530 142851430 695891487 2052350864
33
1651659822 1621900848 479257657 1593414032 1590996924 828907500 529572664 599404747 1394287760 4452580 643571861 1835119275 1182206124 1473439087 855309803 2800373 90123325 1925237767 2073701393 742910421 459277227 556027192 968014844 189842648 576178319 1468548602 1513301028 743973708 235841562 1388644483 41884420 1725369210 1656788691
33
409887077 672228379 1511767509 568262702 1295118583 820932306 191378146 167568332 448780204 645234697 965346896 1311057813 535378631 819229547 98157055 1233440415 1740932733 256325184 1101276573 622520124 1651662376 22978377 979941066 1312283000 1031483275 1351522837 2129605134 880698124 201631029 1575838916 1753487331 865629051 1299438136
