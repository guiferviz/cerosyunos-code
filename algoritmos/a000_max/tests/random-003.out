2133755160
2123302199
1997632515
2146208368
2102335752
2092640521
2118219520
2052350864
2073701393
2129605134
